### DISCUSSION TITLE
Will there be a time oneday when people will be able to download Audacious onto the IPad?

### DISCUSSION THREAD

**Discussion by kikitata87** [audacious-media-player/audacious#1492] [Q&A]
Not really much of a huge question, just wondering.

**Comment by radioactiveman** [comment]
Audacious' focus is to run on PCs and Macs, not on mobile devices. Our developer time is limited and completely voluntary.

So if no one is interested in porting Audacious to Android/iOS independently, this is probably not going to happen.
